packages/ui/src/utils/get-user-updated-properties-schema.ts

```diff
@@ -0,0 +1,41 @@
+import { isDefined } from './is-defined';
+
+export function getUserUpdatedPropertiesSchema(
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  obj1: Record<string, any>,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  obj2: Record<string, any>,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+): Record<string, any> {
+  if (!isDefined(obj1) || !isDefined(obj2)) return {};
+
+  const nonDefaultFormSchema = Object.entries(obj1).reduce(
+    (acc, item) => {
```

Comment:
Having `item` as a parameter of the reducer doesn't help much in knowing what we are working on, would you consider either finding a more suitable name or destructuring the `item` object into `[key, value]` so we don't need to index the `item` array through the code?